packages/blocks/src/__internal__/utils/query.ts

```diff
@@ -256,13 +256,23 @@ export function getStartModelBySelection(range = getCurrentNativeRange()) {
   return startModel;
 }
 
+/**
+ * @deprecated In most cases, you not need RichText, you can use {@link getVirgoEditorByModel} instead.
```

Comment:
```suggestion
 * @deprecated In most cases, you not need RichText, you can use {@link getVirgoByModel} instead.
```
Does this name need to be consistent with the corresponding function name?